README.md

```diff
@@ -1,2 +1,64 @@
-# prometheus-opensearch-dashboards-exporter
+# Prometheus Opensearch Dashboards Exporter
 An exporter for OpenSearch Dashboards
+
+
+## Installing
+With [poetry>=2.0.0](https://python-poetry.org/) installed, clone this repo and run:
+
+```shell
+poetry install
+
+# activate the environment
+eval $(poetry env activate)
```

Comment:
Not sure if this is totally required for running the tool.  Maybe needed for dev?  Not sure.

But also, it's a small enough detail that I'm not going to block on this.